### ISSUE TITLE
Adapting lookup interval in discovery

### ISSUE THREAD

**Issue by kdeme** [status-im/nim-eth#270]
Currently lookups are done at a fixed interval.

When the routing table gets nicely filled up this could be slowed down.

E.g. it could be multiplied with `amount of nodes / max nodes` or something similar.

**Comment by kdeme** [comment]
No longer relevant with after #322 , application can regulate this with launching queries slower/faster
